src/mixins/filing-template-mixin.ts

```diff
@@ -806,16 +815,23 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
    * @param entitySnapshot the latest entity snapshot
    */
   parseChangeRegFiling (filing: ChgRegistrationFilingIF, entitySnapshot: EntitySnapshotIF): void {
+    // make a copy so snapshot objects are different from objects below
+    entitySnapshot = cloneDeep(entitySnapshot)
+
     // store Entity Snapshot
     this.setEntitySnapshot(entitySnapshot)
 
     // store Entity Type
     this.setEntityType(filing.business?.legalType || entitySnapshot.businessInfo.legalType)
 
     // store Business Information
-    this.setBusinessInformation({ ...entitySnapshot.businessInfo })
+    this.setBusinessInformation({
+      ...entitySnapshot.businessInfo,
+      ...filing.business
+    })
```

Comment:
1. Load the initial business info from the snapshot.
2. Override with any business properties in the draft filing.